docs/plugins/advanced_topics/index.md

```diff
@@ -1,20 +1,50 @@
 # Advanced Topics
 
-Advanced topics for plugin developers around the deprecated first generation napari plugins, and how to move to the new system.
+Some plugin developers may find more **in-depth, technical information** helpful. This section describes topics, such as:
+- specifications for napari's plugin ecosystem using the plugin engine `npe2`.
+- migration to `npe2` from the deprecated napari plugin engine v1.
+
+## npe2: napari plugin engine v2
 
 ::::{grid}
-:::{grid-item-card} Migration guide to `npe2`
-:link: npe2-migration-guide
+:::{grid-item-card} npe2 manifest reference
+:link: plugin-manifest
+:link-type: ref
+
+A reference for the `npe2` manifest file, which is used to define the plugin and its contributions.
+
+:::
+
+:::{grid-item-card} `npe2` contributions reference
+:link: contributions-ref
 :link-type: ref
-Have a plugin written for the first generation plugin system? This guide will help you migrate to the new plugin system.
 
+A technical specification for how plugins can contribute additional functionality and features to napari.
+
+:::
+::::
+
+## Deprecated: napari plugin engine v1
+
+:::{important} Use npe2.
```

Comment:
```suggestion
:::{important}
```
If it's `important` it can't have a custom title:
https://sphinx-book-theme.readthedocs.io/en/stable/reference/kitchen-sink/admonitions.html
See also my screenshot here which makes the copy a little pushy:
https://github.com/napari/docs/pull/603#issuecomment-2692960416

Reply:
Fair enough. Thanks! I think that admonition improved things.

Reply:
Does it look different for you? I see:
<img width="330" alt="image" src="https://github.com/user-attachments/assets/d98d3fe4-00ee-4ee1-869b-a9848eb89a68" />